src/foundation/Icon/Generated/IconDotsVertical.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { memo } from "react";
+import { GeneratedIconProps } from "@foundation/Icon/IconProps";
+import { IconSize, IconSizeMap } from "@foundation/Icon/IconSize";
+
+function IconDotsVertical(props: GeneratedIconProps): React.ReactElement<GeneratedIconProps> {
```

Comment:
Can you open a separate PR for adding the Icon? It's not really relevant to the PR and will get lost in the CHANGELOG.